memory-viz/src/memory_model.ts

```diff
@@ -48,10 +59,9 @@ export class MemoryModel {
     list_index_sep: number; // Vertical offset for list index labels
     font_size: number; // Font size, in px
     browser: boolean; // Whether this library is being used in a browser context
-    roughjs_config: object; // Configuration object used to pass in options to rough.js
+    roughjs_config: Config; // Configuration object used to pass in options to rough.js
 
-    constructor(options?: any) {
-        options = options || {};
+    constructor(options: Partial<VisualizationConfig>) {
```

Comment:
Keep this argument optional (and change relevant constructor calls from `MemoryModel({})` back to `MemoryModel()`).